Examples/other_plugins/IDFACollection.swift

```diff
@@ -77,6 +74,17 @@ class IDFACollection: Plugin {
     }
 }
 
+extension IDFACollection: iOSLifecycle {
+    func applicationDidBecomeActive(application: UIApplication?) {
+        let status = ATTrackingManager.trackingAuthorizationStatus
+        if status == .notDetermined && !alreadyAsked {
+            // we don't know, so should ask the user.
+            askForPermission()
+            alreadyAsked = true
```

Comment:
lets move this above askForPermission().

Reply:
@bsneed so remove the new extension, add `iOSLifecycle` protocol to existing `extension IDFA` as well?